pysyncobj/journal.py

```diff
@@ -105,6 +122,27 @@ def flush(self):
         self.__mm.flush()
 
 
+class MetaStorer(object):
+    def __init__(self, path):
+        self.__path = path
+
+    def getMeta(self):
+        meta = {}
+        try:
+            meta = loads(open(self.__path, 'rb').read())
+        except:
+            pass
+        return meta
+
+    def storeMeta(self, meta):
+        with open(self.__path + '.tmp', 'wb') as f:
+            f.write(dumps(meta))
+            f.flush()
+        os.rename(self.__path + '.tmp', self.__path)
```

Comment:
On windows it fails to overwrite the already existing file:
```python
FileExistsError: [WinError 183] Cannot create a file when that file already exists:
```

AFAIK it is a known issue and therefore recommended to use `shutil.move()` for compatibility.
Although, internally it also first calls os.rename() and on failure (because the destination file already exists) copying the content and stats (more work).

Reply:
Thanks, fixed.